src/transformData.js

```diff
@@ -0,0 +1,21 @@
+function TransformEventData (componentData) {
```

Comment:
I believe the convention for functions is for them to be in camelCase, instead of PascalCase

Reply:
Function names has been changed to camelCase.